src/coreComponents/physicsSolvers/fluidFlow/SourceFluxStatistics.cpp

```diff
@@ -210,21 +243,36 @@ bool SourceFluxStatsAggregator::execute( real64 const GEOS_UNUSED_PARAM( time_n
           subRegionStats.finalizePeriod();
           regionStats.stats().combine( subRegionStats.stats() );
         } );
-
         fluxStats.stats().combine( regionStats.stats() );
-        writeStatsToLog( 3, region.getName(), regionStats );
-        writeStatsToCSV( region.getName(), regionStats, false );
+
+        gatherStatsForLog( isLogLevelActive< logInfo::sourceFluxRegionsStatistics >( getLogLevel() ),
+                           region.getName(), tableRegionsData, regionStats );
```

Comment:
What do you think of this shorter template:
```suggestion
        gatherStatsForLog< allSourceFluxesStats >( region.getName(), tableRegionsData, regionStats );
```
It is a private method, so you can define this template in the cpp. A method-local `using` can be used too.

Reply:
If you prefer not to touch the signature of `gatherStatsForLog()`, you can just define 3 boolean:
```
bool const allStats = isLogLevelActive< logInfo::sourceFluxRegionsStatistics >( getLogLevel() );
bool const fluxesStats = isLogLevelActive< logInfo::sourceFluxStatistics >( getLogLevel() );
bool const regionsStats = isLogLevelActive< logInfo::sourceFluxRegionsStatistics >( getLogLevel() );
```